Gear/Detonators.md

```diff
@@ -5,7 +5,8 @@ To use a detonator, right click while holding one and it will be thrown in the d
 
 Detonators are relatively expensive, so it is wise to use them sparingly and not carry too many, lest you die and lose them all.
 
-## Crafting
+**Crafting**
```

Comment:
This was supposed to be how it was

Reply:
Replaced that formatting to original state.